src/components/App/App.js

```diff
@@ -102,19 +117,13 @@ class App extends Component {
     return (
       <>
         {!as.viewingAllAnnotations && !as.viewingAllPredictions && (
-          <Block
-            key={(as.selectedHistory ?? as.selected)?.id}
-            name="main-view"
-            onScrollCapture={this._notifyScroll}
-          >
```

Comment:
it seems that your editor does some unnecessary formatting. please, check your  the settings